src/fidesops/util/logger.py

```diff
@@ -42,11 +43,14 @@ def factory(  # pylint: disable=R0913
     return factory
 
 
-def _mask_pii_for_logs(pii_text: Any) -> Any:
+def _mask_pii_for_logs(parameter: Any) -> Any:
     """
-    :param pii_text: param that contains possible pii
-    :return: depending on ENV config, returns masked pii param
+    :param parameter: param that contains possible pii
+    :return: depending on ENV config, returns masked pii param.
+
+    Don't mask numeric values as this can throw errors in consumers
+    format strings.
     """
-    if isinstance(pii_text, NotPii):
-        return pii_text
+    if isinstance(parameter, (NotPii, Number)):
```

Comment:
This looks good for now, but I wonder about the case where PII could eventually be a number. A credit card number for instance would be a good example here, and currently that wouldn't be masked.

Reply:
Agreed, with a few caveats - 
A CC should never be a number, since it can get mangled if it starts with an '0'. That being said, people do weird things in databases. It's also possible that something like an int ID might be PII since it could identify you in another system.

Ultimately this was a solution I came up with to quick-patch the issue. I think the _correct_ solution would be to switch from whitelist to blacklist - wouldn't really be much more code for us, and would not affect any embedded libraries like whitelist does. That's a bigger fix, but I think we should put it on the board.